VikiGolieusova/src/ContactList/ContactListTest.java

```diff
@@ -0,0 +1,69 @@
+package ContactList;
+
+public class ContactListTest {
```

Comment:
init test data before each test
you should extract every test to a separate method
thereafter you can init data before test logic will be called